test/js/inline-style-property-map.js

```diff
@@ -21,6 +21,16 @@ suite('Inline StylePropertyMap', function() {
     assert.strictEqual(this.element.style['height'], simpleLength.cssString);
   });
 
+  test('The set method successfully sets the CSS string of the list of StyleValues on property ' +
+    'if it supports setting lists of values', function() {
```

Comment:
For brevity:
"Set successfully sets the style string for the list of StyleValues on a property that supports sequences."